client/src/views/Tools.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <section>
+    <the-title title="Tools" icon="hammer-screwdriver" />
```

Comment:
Please add a top margin so it would have the same spacing as Games page for example

Reply:
It can be inner feature in TheTitle component, so if you don't have button, then you have spacing